src/common/components/dropdown/dropdown.component.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import classes from './dropdown.component.module.css';
+import { OptionGroup } from './components/option-group.component';
+import { ExpandDown } from '../icons/expand-down-icon.component';
+import { DropdownOptionVm } from './dropdown.model';
+
+interface Props {
+  name: string;
+  options: DropdownOptionVm[];
+  label?: string;
+  onKeySelected: (field: DropdownOptionVm) => void;
+  selectTitle?: string;
+  selectedField?: DropdownOptionVm;
+}
+
+export const Dropdown: React.FC<Props> = props => {
+  const { name, label, options, onKeySelected, selectedField, selectTitle } =
+    props;
+
+  const [selectedPath, setSelectedPath] = React.useState(selectedField?.label);
+  const [optionsListVisible, setOptionsListVisible] = React.useState(false);
+  const [currentSelectedKeyFieldId, setCurrentSelectedKeyFieldId] =
+    React.useState(selectedField?.id);
+
+  const handleOptionClick = (option: DropdownOptionVm) => {
+    setCurrentSelectedKeyFieldId(option.id);
+    setSelectedPath(option.label);
+    setOptionsListVisible(false);
+    onKeySelected(option);
+  };
+
+  return (
+    <>
+      <div className={classes.select}>
+        <p className={classes.selectLabel}>{label ? label : ''}</p>
+        <div className={classes.selectSelect}>
+          <div
+            className={classes.selectChosen}
+            onClick={() => setOptionsListVisible(!optionsListVisible)}
+          >
+            <p className={classes.selectText}>
+              {selectedPath || selectTitle || 'Selecciona una opción'}
```

Comment:
'Seleccion una option', should be enlgish "Select an option"

And we should place it outside the component in a const, something like

```ts
const SELECT_AN_OPTION = "Select an option";
```

then use it

```diff
            <p className={classes.selectText}>
-              {selectedPath || selectTitle || 'Selecciona una opción'}
+              {selectedPath || selectTitle || SELECT_AN_OPTION}
```

Reply:
It's fixed.